extension/src/experiments/columns/collect/index.test.ts

```diff
@@ -28,106 +32,91 @@ describe('collectColumns', () => {
   })
 
   it('should output both params and metrics when both are present', () => {
-    const columns = collectColumns({
-      [EXPERIMENT_WORKSPACE_ID]: {
-        baseline: {
-          data: {
-            metrics: {
-              1: {
-                data: { 2: 3 }
-              }
-            },
-            params: {
-              a: {
-                data: {
-                  b: 'c'
-                }
-              }
+    const columns = collectColumns(
+      generateWorkspaceOnlyExpShowOutput({
+        metrics: {
+          1: {
+            data: { 2: 3 }
+          }
+        },
+        params: {
+          a: {
+            data: {
+              b: 'c'
             }
           }
         }
-      }
-    })
+      })
+    )
     const params = columns.find(column => column.type === ColumnType.PARAMS)
     const metrics = columns.find(column => column.type === ColumnType.METRICS)
     expect(params).toBeDefined()
     expect(metrics).toBeDefined()
   })
 
   it('should omit params when none exist in the source data', () => {
-    const columns = collectColumns({
-      [EXPERIMENT_WORKSPACE_ID]: {
-        baseline: {
-          data: {
-            metrics: {
-              1: {
-                data: { 2: 3 }
-              }
-            }
+    const columns = collectColumns(
+      generateWorkspaceOnlyExpShowOutput({
+        metrics: {
+          1: {
+            data: { 2: 3 }
           }
         }
-      }
-    })
+      })
+    )
     const params = columns.find(column => column.type === ColumnType.PARAMS)
     const metrics = columns.find(column => column.type === ColumnType.METRICS)
     expect(params).toBeUndefined()
     expect(metrics).toBeDefined()
   })
 
   it('should return an empty array if no params and metrics are provided', () => {
-    const columns = collectColumns({
-      [EXPERIMENT_WORKSPACE_ID]: {
-        baseline: {}
-      }
-    })
+    const columns = collectColumns(generateWorkspaceOnlyExpShowOutput())
     expect(columns).toStrictEqual([])
   })
 
   const exampleBigNumber = 3000000000
-  const columns = collectColumns({
-    branchA: {
-      baseline: {
-        data: {
-          params: {
-            'params.yaml': {
-              data: { mixedParam: 'string' }
-            }
-          }
-        }
-      },
-      otherExp: {
-        data: {
-          params: {
-            'params.yaml': {
-              data: { mixedParam: true }
-            }
-          }
+  const data: ExpShowOutput = [
```

Comment:
[I] Creation of data has to be completely wrapped in these util as otherwise another "overhaul" could lead to another round of manual work